src/main/java/wrup/spring/a_wstrzykiwanie_zależności/Samochód.java

```diff
@@ -0,0 +1,25 @@
+package wrup.spring.a_wstrzykiwanie_zależności;
+
+import wrup.spring.a_wstrzykiwanie_zależności.silnik.Silnik;
+
+/**
+ * Ziarno springowe, do którego wstrzykiwany zostaje Silnik. {@link Silnik}
```

Comment:
Niezgodność czasów: imiesłów wstrzykiwany sugeruje przyszły a na pewno niedokonany, zostaje to z kolei teraźniejszy? Zamiast tego: wstrzyknięty zostanie (przyszły dokonany) albo wstrzykiwany będzie (niedokonany przyszły)?